monitoring/base/victoriametrics/rules/sabakan-alertrule.yaml

```
@@ -31,7 +31,7 @@ spec:
             summary: "Not-healthy boot server(s) exist."
             runbook: TBD
           expr: |
-            sum(sabakan_machine_status{status!="healthy", role="boot"}) > 0
+            sum(sabakan_machine_status{status!="healthy", status!="retired", role="boot"}) > 0
```

Why don't you exclude retiring?

```suggestion
            sum(sabakan_machine_status{status!~"healthy|retiring|retired", role="boot"}) > 0
```

---

I've communicated elsewhere, but I'm writing this for the record.
Since retiring is originally expected to automatically transition to retired, I thought it was an anomaly for the retiring state to continue.
The use of regular expressions to get metrics has been reflected.